google/cloud/pubsub/internal/streaming_subscription_batch_source.cc

```diff
@@ -152,17 +176,30 @@ future<Status> StreamingSubscriptionBatchSource::BulkNack(
   auto requests =
       SplitModifyAckDeadline(std::move(request), kMaxAckIdsPerMessage);
   if (requests.size() == 1) {
```

Comment:
Aside: Is it worth making a special case of `size() == 1`?

Reply:
Not sure... don't have strong opinions